detox/src/devices/drivers/android/genycloud/GenyCloudDriver.js

```diff
@@ -16,6 +16,7 @@ const InstanceLifecycleService = require('./services/GenyInstanceLifecycleServic
 const InstanceLookupService = require('./services/GenyInstanceLookupService');
 const InstanceNaming = require('./services/GenyInstanceNaming');
 const RecipesService = require('./services/GenyRecipesService');
+const DetoxGenymotionManager = require('../../../../android/espressoapi/DetoxGenymotionManager');
```

Comment:
Looks fishy a bit, but okay.

Reply:
I agree, but it's mostly about naming. The approach is good.